x-hooks/use-authentication.tsx

```diff
@@ -185,15 +186,14 @@ export function useAuthentication() {
       state.Service.active.getOraclesResume(state.currentUser.walletAddress),
 
       state.Service.active.getBalance('settler', state.currentUser.walletAddress),
-      state.Service.active.getBalance('eth', state.currentUser.walletAddress),
-      state.Service.active.getBalance('staked', state.currentUser.walletAddress),
-
+      searchCurators({ address: state.currentUser.walletAddress, networkName: state.Service?.network?.active?.name })
```

Comment:
it would be easier to have a `getTotalStaked` function on the `useApi()` context